server/platform/services/docextractor/archive.go

```diff
@@ -23,8 +25,8 @@ func (ae *archiveExtractor) Name() string {
 }
 
 func (ae *archiveExtractor) Match(filename string) bool {
-	_, err := archiver.ByExtension(filename)
-	return err == nil
+	format, _, err := archives.Identify(context.Background(), filename, nil)
+	return err == nil && format != nil
```

Comment:
The check on `format` seems redundant. Could we skip it?

> If no matching formats were found, special error NoMatch is returned.

https://pkg.go.dev/github.com/mholt/archives#Identify